books/RayTracingInOneWeekend.html

```diff
@@ -2779,6 +2782,239 @@
 
 
 
+Creating a Scene
+====================================================================================================
+Sometimes you just have to make a scene. At this point, our `main.cc` has a standalone function to
```

Comment:
cut "Sometimes you just have to make a scene."

Reply:
Awwwww.....

Reply:
Fun while it lasted.

Reply:
If you want to keep it, just move it down